lib/static_data/championship_eligible_iso2.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I don't see this anywhere in the db? But it looks simple enough that I trust it

Reply:
The table name in the DB is much longer. But I am confident that the data is correct.

Reply:
For reference, the table is called `eligible_country_iso2s_for_championship`